bsi_zoo/cross_val.py

```diff
@@ -0,0 +1,194 @@
+from scipy import linalg
+import numpy as np
+from sklearn.model_selection import GridSearchCV
+from sklearn.base import BaseEstimator, ClassifierMixin, clone
+from sklearn.model_selection import check_cv
```

Comment:
@AliHashemi-ai I moved all the cross-val code to one file

Reply:
Sounds great Thank you